package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "build": "turbo run build --include-dependencies --no-deps --scope=*@aws-amplify/*",
+    "build": "turbo run build --filter=./packages/* --filter=!./packages/angular",
```

Comment:
This is sufficient condition for now. We only build what's on `packages/*` that is not the `ui-angular-mono` ones. The other packages do not have `build` commands so they're automatically excluded. 

Opinions welcome!